snap/local/reset_config.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+# Make sure the folder exists
+mkdir -p $SNAP_COMMON/config
+
+cp $SNAP/usr/share/rosbot-xl-teleop/config/*.yaml $SNAP_COMMON/config/.
```

Comment:
```suggestion
cp $SNAP/usr/share/rosbot-xl-teleop/config/*.yaml $SNAP_COMMON/config/
```
Is that dot necessary?

Reply:
It is not. Just a convention to mean "this directory".